hack/multi-node/user-data.sample

```diff
@@ -19,36 +19,58 @@ coreos:
           [Unit]
           Requires=flanneld.service
           After=flanneld.service
+    - name: kubelet-initial-config.service
+      enable: true
+      command: start
+      content: |
+        [Service]
+        ExecStart=/home/core/install-initial-kubelet-config
```

Comment:
The ordering doesn't seem to work here, gave it a shot and it complained about the EnvironmentFile missing.